stdlib/rubygems/0/version.rbs

```diff
@@ -170,7 +170,8 @@ module Gem
     #     ver2 = Version.create(ver1)       # -> (ver1)
     #     ver3 = Version.create(nil)        # -> nil
     #
-    def self.create: (_ToS | Version | nil input) -> instance?
+    def self.create: (_ToS | Version input) -> instance
+                   | (nil input) -> nil
```

Comment:
[note] The test case is covered when `nil` is passed:

https://github.com/ruby/rbs/blob/e629aa7d9f5d55bcbba6632e598835a6651bc32c/test/stdlib/rubygems/GemVersion_test.rb#L23-L24